### PR TITLE
Color Picker slider updates

### PR DESCRIPTION
**Pull request by leonMSFT**
- Round out the left/right edges of the sliders a bit
- Add a small filled circle in the center of the thumb
- Give the thumb a slight border

SliderLight
SliderDark

### CODE REVIEW

**Comment by robloo** [comment]
Looks familiar! (https://github.com/windows-toolkit/WindowsCommunityToolkit/issues/3363)

Right now I would only recommend to the design team that the height of the slider itself should be increased to show more color and make it a better target for touch interaction.

**Comment by gabrielconl** [comment]
The slider bar is 11px tall and the thumb is 20px, making them slightly misaligned (it should be an even number). Maybe the space between the two sliders could also be decreased.

File: dev/ColorPicker/ColorPicker.xaml [hunk 1]
```diff
@@ -26,12 +26,22 @@
...
                                                                 <ControlTemplate TargetType="Thumb">
-                                                                    <Border BorderBrush="{TemplateBinding BorderBrush}" BorderThickness="{TemplateBinding BorderThickness}" Background="{TemplateBinding Background}" CornerRadius="{ThemeResource SliderThumbCornerRadius}" />
+                                                                    <Border BorderBrush="{ThemeResource SliderThumbBorderBrush}"
+                                                                            BorderThickness="{TemplateBinding BorderThickness}"
+                                                                            Background="{ThemeResource SliderOuterThumbBackground}"
+                                                                            CornerRadius="{ThemeResource SliderThumbCornerRadius}">
+                                                                        <Ellipse
+                                                                            x:Name="SliderInnerThumb"
+                                                                            Fill="{TemplateBinding Background}"
```
Comment (ranjeshj) [inline dev/ColorPicker/ColorPicker.xaml:40]: >                                                   [](start = 111, length = 50)

nit: stray spaces.

**Review by ranjeshj — APPROVED** [review]

**Pull request merged**